docs/topics/time.md

```diff
@@ -0,0 +1,373 @@
+Enum class:
+  * `NANOSECONDS`
+  * `MICROSECONDS`
+  * `MILLISECONDS`
+  * `SECONDS`
+  * `MINUTES`
+  * `HOURS`
+  * `DAYS`
+
+`Durations` can also be zero, negative or infinite.
+
+### Calculate duration
+
+To create an instance of the `Duration` class, use the [extension properties](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/#companion-object-properties) 
+available for `Int`, `Long`, and `Double` types: `nanoseconds`, `microseconds`, `milliseconds`, `seconds`, `minutes`, 
+`hours`, and `days`.
+
+For example:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.Duration.Companion.minutes
+
+fun main() {
+    //sampleStart
+    val fiveHundredMilliseconds: Duration = 500.milliseconds
+    val fiveSeconds: Duration = 5.seconds
+    val tenMinutes: Duration = 10.minutes
+
+    println(fiveHundredMilliseconds) // 500ms
+    println(fiveSeconds)             // 5s
+    println(tenMinutes)              // 10m
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+To create a `Duration` of zero seconds, use the `ZERO` property:
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.ZERO
+
+fun main() {
+    //sampleStart
+    val zero: Duration = ZERO
+
+    println(zero) // 0s
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+You can also perform basic arithmetic with `Durations`:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.seconds
+
+fun main() {
+    //sampleStart
+    val fiveSeconds: Duration = 5.seconds
+    val thirtySeconds: Duration = 30.seconds
+
+    println(fiveSeconds + thirtySeconds)
+    // 35s
+    println(thirtySeconds -  fiveSeconds)
+    // 25s
+    println(fiveSeconds * 2)
+    // 10s
+    println(thirtySeconds / 2)
+    // 15s
+    println(thirtySeconds / fiveSeconds)
+    // 6.0
+    println(-thirtySeconds)
+    // -30s
+    println((-thirtySeconds).absoluteValue)
+    // 30s
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+### Print duration
+
+To print a `Duration`, you have already seen that you can use `println()` and `print()` functions. By default, the
+time is reported using each unit that is present. For example: `1h 0m 45.677s`.
+
+To configure the output, use the `.toString()` function with your desired `Duration.Unit` and number of decimal places
+as function parameters:
+
+```kotlin
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.DurationUnit
+
+fun main() {
+    //sampleStart
+    val fiveSeconds: Duration = 5.seconds
+    
+    // Print in minutes with 2 decimal places
+    println(fiveSeconds.toString(DurationUnit.MINUTES, 2))
+    // 0,08m
```

Comment:
Better to use a duration with some sub-second part and format it to seconds for example:
```
5887.milliseconds.toString(DurationUnit.SECONDS, 2) // 5.89s
```